src/components/slides/EventSlide/EventChart/EventChart.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { useD3 } from "../../../../hook/useD3";
+import * as d3 from "d3";
+import "./EventChart.css";
+import PropTypes from "prop-types";
+
+export const ExampleofLine = ( { data } ) => {
+    const ref = useD3(
+        ( svg ) => {
+            const height = 500;
+            const width = 1150;
+            const margin = { top: 20, right: 30, bottom: 30, left: 60 };
+            
+            var x = d3
+                .scaleBand()
+                .domain( data.map( d => {return d.year; } ) )
+                .range( [ 0, width - margin.left ] )
+                .padding( 0.2 );
+
+            var y = d3
+                .scaleLinear()
+                .domain( [ 0, d3.max( data, ( d ) => d.sales ) ] )
+                .rangeRound( [ height, margin.top ] );
+  
+            svg.append( "g" )
+                .call( d3.axisBottom( x ) )
+                .attr( "transform", "translate(" + margin.left + "," + ( height + height / 20 ) + ")" )
+                .selectAll( "text" );
+
+            svg.append( "g" )
+                .call( d3.axisLeft( y ) )
+                .attr( "transform", "translate(" + margin.left + "," + ( height / 20 ) + ")" );
+
+            // Add the line
+            svg.append( "path" )
+                .datum( data )
+                .attr( "fill", "none" )
+                .attr( "stroke", "steelblue" )
+                .attr( "stroke-width", 1.5 )
+                .attr( "transform", "translate(" + margin.left + ",0)" )
+                .attr( "d", d3.line()
+                    .x( function( d ) { return x( d.year ); } )
```

Comment:
```suggestion
                    .x( ( d ) => { return x( d.year ); } )
```

Reply:
This is just to try to be consistent with the rest of the app, we define all the function as : const functionName = (params) => {logic to do}